src/apiCalls.js

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+
+const url = axios.create({
+  baseURL: "https://nc-news-y5oc.onrender.com/api",
+});
+
+export function getAllArticles() {
+  return url.get("/articles").then(({ data }) => {
+    return data;
+  });
+}
+
+export function getArticleById(article_id) {
+  return url.get(`/articles/${article_id}`).then(({ data }) => {
+    return data;
+  });
+}
+
+export function getCommentsById(article_id) {
+  return url.get(`/articles/${article_id}/comments`).then(({ data }) => {
+    return data;
+  });
+}
+
+export function upvote(article_id) {
+  return url.patch(`/articles/${article_id}`, { inc_votes: 1 }).then((data) => {
+    return data;
+  });
+}
+
+export function downvote(article_id) {
+  return url.patch(`/articles/${article_id}`, { inc_votes: -1 }).then((data) => {
+    return data;
+  });
+}
```

Comment:
This is not DRY. We shouldn't have multiple API functions that use the same API endpoint is should instead be one function that is dynamic.